docs/changelog.md

```diff
@@ -4,13 +4,13 @@ title: Changelog
 permalink: /changelog/
 ---
 
-# 115.0.0 (In Development)
+# 115.0 (In Development)
```

Comment:
I noticed starting on 110.0 that we basically dropped the extra `.0` to the versioning https://maven.mozilla.org/?prefix=maven2/org/mozilla/components/compose-tabstray/